llamagon_ocr_script/main.py

```diff
@@ -32,7 +30,13 @@ def main(file_name, upload):
     parser.add_argument(
         "--upload", action="store_true", help="Upload the result folder to the cloud."
     )
+    parser.add_argument(
+        "--input_folder", type=str, default="input", help="The input folder containing the PDF file."
+    )
+    parser.add_argument(
+        "--output_folder", type=str, default="output", help="The folder to save the output images and text."
+    )
```

Comment:
where is the file_name parser? You used `args.file_name` but didn't have an argument for it.